server/src/test/java/org/elasticsearch/common/RoundingTests.java

```diff
@@ -778,6 +803,18 @@ public void testPrepareLongRangeRoundsNotToMidnight() {
         assertThat(prepared.round(time("9000-03-31T15:25:15.148Z")), isDate(time("9000-03-31T15:00:00Z"), tz));
     }
 
+    public void testIntervalBeforeGap() {
```

Comment:
These were fun examples that came up in the random tests that failed. It makes it a ton easier to rerun them if I pull them out like this.